joomla/joomla-cms
# [a11y] change ordered list in blog-links to unordered list

Pull Request for Issue this issue .

### Summary of Changes
This PR will change the unordered list of com_content > category > blog-links.php into an unordered list.
The list is no sequential information but a list of items where the order is not relevant. 
Therefor an unordered list should be used. 

`<ul></ul>` instead of `<ol></ol>`


### Testing Instructions

- Joomla test environment with demo sample data. 
- Go to Sample Layouts > Blog
- Scroll to the pagination and notice a single link with title = _Welcome to your blog_

### Actual result BEFORE applying this Pull Request

![Scherm­afbeelding 2023-05-15 om 16 05 18](https://github.com/joomla/joomla-cms/assets/639822/14a6f69a-4495-43f7-8341-cbc811f063f4)

### Expected result AFTER applying this Pull Request

![Scherm­afbeelding 2023-05-15 om 16 05 37](https://github.com/joomla/joomla-cms/assets/639822/7b6774f7-12e4-47b4-910e-165b411f2a42)


### Link to documentations
Please select:
